human_aware_rl/rllib/rllib.py

```diff
@@ -622,7 +622,7 @@ def save_trainer(trainer, params, path=None):
         dill.dump(config, f)
     return save_path
 
-def load_trainer(save_path):
+def load_trainer(save_path, true_num_workers=False):
```

Comment:
What is `true_num_workers`? Why do we need it for loading the trainer? We should probably add documentation to the docstring, or below (where it's being used in the if statement)